### PR TITLE
Priority Hints article

### PR DESCRIPTION
**Pull request by yoavweiss**
What's changed, or what was fixed?
- We wrote an article about Priority Hints

**Target Live Date:** YYYY-MM-DD

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2019/02/priority-hints.md [hunk 1]
```diff
@@ -0,0 +1,432 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+
+{# wf_updated_on: 2019-02-14 #}
+{# wf_published_on: 2019-02-14 #}
+{# wf_tags: fundamentals, performance #}
+{# wf_featured_image: /web/updates/images/2019/02/priority-hints/image1.png #}
+{# wf_featured_snippet: Priority Hints are coming to an Origin Trial near you! Try them out! #}
+{# wf_blink_components: N/A #}
```
Comment (tomayac) [inline src/content/en/updates/2019/02/priority-hints.md]: Blink>Loader

File: src/content/en/updates/2019/02/priority-hints.md [hunk 2]
```diff
@@ -0,0 +1,432 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+
```
Comment (tomayac) [inline src/content/en/updates/2019/02/priority-hints.md]: Add:

description: Priority Hints provide developers a way to indicate a resource's relative importance to the browser, allowing more control over the order resources are loaded.

**Comment by yoavweiss** [comment]
OK, so my contributor image is broken for some reason I can't quite explain, the "did you find it useful?" widget seems equally broken and the linter is mad at me for having `<script>` in the examples.

Otherwise, I think this is ready for review.
@tomayac @addyosmani @samdutton - You already reviewed the article in its doc format. Care to take a look here?

**Comment by tomayac** [comment]
> OK, so my contributor image is broken for some reason I can't quite explain

It looks fine to me.

> [T]he "did you find it useful?" widget seems equally broken

This is just in staging.

> [T]he linter is mad at me for having `<script>` in the examples.

This is OK to ignore.

> @tomayac @addyosmani @samdutton - You already reviewed the article in its doc format. Care to take a look here?

LGTM with the nits in the review above.

File: src/content/en/updates/2019/02/priority-hints.md (hunk 1 above)
Comment (yoavweiss) [reply src/content/en/updates/2019/02/priority-hints.md]: cool

**Pull request merged**